src/utils/lazyLoad.ts

```diff
@@ -1,45 +0,0 @@
-/*---------------------------------------------------------------------------------------------

```

Comment:
It seems we never needed this complicated pattern for lazy loading, it Just Works (TM) without all the complexity.